ie-comunicaciones.csl

```diff
@@ -0,0 +1,205 @@
+  <info>
+    <title>IE Comunicaciones</title>
+    <title-short>IE Com</title-short>
+    <id>http://www.zotero.org/styles/ie-comunicaciones</id>
+    <link href="http://www.zotero.org/styles/ie-comunicaciones" rel="self"/>
+    <link href="http://www.zotero.org/styles/harvard-european-archaeology" rel="template"/>
+    <link href="http://dialnet.unirioja.es/servlet/revista?info=nor_estilo&amp;codigo=12765&amp;modo=popup" rel="documentation"/>
+    <author>
+      <name>Hendrys Tobar</name>
+      <email>u1918137@campus.udg.edu</email>
+      <uri>http://www.mendeley.com/profiles/hendrys-tobar/</uri>
+    </author>
+    <category citation-format="author-date"/>
+    <issn>1699-4574</issn>
+    <summary>Revista IE Comunicaciones. </summary>
+    <updated>2015-02-25T12:29:51+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="volume" form="symbol">V</term>
+      <term name="issue" form="symbol">N</term>
+    </terms>
+  </locale>
+  <citation et-al-min="2" et-al-use-first="1" et-al-subsequent-min="2" et-al-subsequent-use-first="1" disambiguate-add-names="true" disambiguate-add-givenname="true" disambiguate-add-year-suffix="true">
+    <layout delimiter=", " prefix="[" suffix="]">
+      <group delimiter=", ">
+        <group delimiter=" ">
+          <text macro="author-short"/>
+          <text macro="year-date"/>
+        </group>
+      </group>
+    </layout>
+  </citation>
+  <macro name="editor">
+    <names variable="editor" delimiter=", ">
+      <name and="text" initialize-with=". "/>
+      <label form="short" prefix=" (" suffix=")" strip-periods="true"/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text term="anonymous" form="short" text-case="capitalize-first" strip-periods="true"/>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="10" et-al-use-first="10" et-al-subsequent-min="10" et-al-subsequent-use-first="10" initialize-with="."/>
+      <label form="short" prefix=" "/>
+      <substitute>
+        <names variable="editor"/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="2" et-al-use-first="1" et-al-subsequent-min="2" et-al-subsequent-use-first="1" initialize-with=". "/>
+      <substitute>
+        <names variable="editor"/>
```

Comment:
I don't understand what yu're saying. The substitute element as I understand it (it was already in the template used) is there to replace the name is none is given. It replaces with the editor or the translator whichever is present.

I'm using it in the bibliography because it contains the same as in the inline citation.

Please clarify what youy are trying to tell me. thanks.